lib/grpc/server.ex

```diff
@@ -214,21 +218,21 @@ defmodule GRPC.Server do
   #
   #   * `:cred` - a credential created by functions of `GRPC.Credential`,
   #               an insecure server will be created without this option
-  #   * `:adapter` - use a custom server adapter instead of default `GRPC.Adapter.Cowboy`
+  #   * `:adapter` - use a custom server adapter instead of default `GRPC.Server.Adapters.Cowboy`
   @doc false
-  @spec start(servers_list, non_neg_integer, Keyword.t()) :: {atom, any, non_neg_integer}
+  @spec start(servers_list, server_port, Keyword.t()) :: {atom, any, non_neg_integer}
   def start(servers, port, opts \\ []) do
-    adapter = Keyword.get(opts, :adapter, GRPC.Adapter.Cowboy)
+    adapter = Keyword.get(opts, :adapter, GRPC.Server.Adapters.Cowboy)
     servers = GRPC.Server.servers_to_map(servers)
     adapter.start(nil, servers, port, opts)
   end
 
   @doc false
-  @spec start_endpoint(atom, non_neg_integer, Keyword.t()) :: {atom, any, non_neg_integer}
+  @spec start_endpoint(endpoint, server_port, Keyword.t()) :: {atom, any, non_neg_integer}
   def start_endpoint(endpoint, port, opts \\ []) do
     servers = endpoint.__meta__(:servers)
     servers = GRPC.Server.servers_to_map(servers)
-    adapter = Keyword.get(opts, :adapter, GRPC.Adapter.Cowboy)
+    adapter = Keyword.get(opts, :adapter, GRPC.Server.Adapters.Cowboy)
```

Comment:
Could you replace all of these Keyword.get(a, b, c) with Keyword.get(a, b) || c?

Reply:
Just the ones this PR touches is fine, though